matchbox_socket/Cargo.toml

```diff
@@ -26,7 +26,7 @@ log = { version = "0.4", default-features = false }
 thiserror = "1.0"
 cfg-if = "1.0"
 async-trait = { version = "0.1.64" }
-once_cell = { version = "1.17", default-features = false }
+once_cell = { version = "1.17", default-features = false, features = ["race", "alloc"] }
```

Comment:
I didn't see where these features or once_cell is used. Did I miss it?

Reply:
It's a drive by fix. I think we don't run into it in the examples because other dependencies enable those feature.

I ran into it when invoking clippy for wasm from the project root

Reply:
Was added in #140